install-scripts/install-debian.sh

```diff
@@ -0,0 +1,40 @@
+# This installation has been tested on Debian 11 XFCE
+
+# Install curl in order to install Rust and Cargo
+# Install make, necessary for installing python 3.9 with pyenv
+
+sudo apt install -y curl 
+                    \ make
+
+# Install Rust and Cargo
+curl https://sh.rustup.rs -sSf | sh
```

Comment:
For automation, we need all installations to be non-interactive.

Does Rust installation have any interactive steps in which you have to say yes?

Reply:
Also: do we have to source some files to have `rustc` available in our PATH?